Nunit_proj/Pages/CertificationNegativepage.cs

```diff
@@ -0,0 +1,88 @@
+﻿using Nunit_proj.Utilities;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Nunit_proj.Pages
+{
+    public class CertificationNegativepage : CommonDriver
```

Comment:
you should avoid creating pages by test cases e.g happy or negative! your page should only be named around the actual feature. for future please rename this accordingly for now its okay